modules/claims_api/spec/requests/v1/forms/526_spec.rb

```diff
@@ -2310,34 +2322,45 @@ def obj.class
       context "when 'disabilites.secondaryDisabilities.classificationCode' does not match name" do
         let(:classification_type_codes) { [{ clsfcn_id: '1111' }] }
 
-        before do
-          expect_any_instance_of(BGS::StandardDataService)
-            .to receive(:get_contention_classification_type_code_list).and_return(classification_type_codes)
-        end
+        [true, false].each do |flipped|
+          context "when feature flag is #{flipped}" do
+            before do
+              if flipped
+                Flipper.enable(:claims_api_526_validations_v1_local_bgs)
```

Comment:
@rjohnson2011 Just one question for clarification.  Do you want me to avoid running basically the tests twice with the flipper on/off, or would updating each test to use the bool to return on the flipper be acceptable?  Just want to make sure to dial it in correctly here.

To be clear, what I am asking is, is the change you want just to remove the `Flipper.enable(:feature_flag)` at the top and instead just pass the bool value to that `allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(flipped)` line in the individual tests?

Thanks!